docs/user/image_mode.md

```diff
@@ -0,0 +1,517 @@
+```
+
+> Replace `<myreg>` with the URL to your remote registry, and `<myorg>/<mypath>`
+> with your organization name and path inside your remote registry.
+
+> See the `containers-auth.json(5)` manual pages for more information on the
+> syntax of the `AUTH_CONFIG` registry authentication file.
+
+Run the following commands to create the `kickstart.ks` file to be used during
+the virtual machine installation.
+
+```bash
+cat > kickstart.ks <<EOFKS
+lang en_US.UTF-8
+keyboard us
+timezone UTC
+text
+reboot
+
+# Partition the disk with hardware-specific boot and swap partitions, adding an
+# LVM volume that contains a 10GB+ system root. The remainder of the volume will
+# be used by the CSI driver for storing data.
+zerombr
+clearpart --all --initlabel
+# Create boot and swap partitions as required by the current hardware platform
+reqpart --add-boot
+# Add an LVM volume group and allocate a system root logical volume
+part pv.01 --grow
+volgroup rhel pv.01
+logvol / --vgname=rhel --fstype=xfs --size=10240 --name=root
+
+# Lock root user account
+rootpw --lock
+
+# Configure network to use DHCP and activate on boot
+network --bootproto=dhcp --device=link --activate --onboot=on
+
+%pre-install --log=/dev/console --erroronfail
+
+# Create a 'bootc' image registry authentication file
+mkdir -p /etc/ostree
+cat > /etc/ostree/auth.json <<'EOF'
+$(cat "${AUTH_CONFIG}")
+EOF
+
+%end
+
+# Pull a 'bootc' image from a remote registry
+ostreecontainer --url "${IMAGE_REF}"
+
+%post --log=/dev/console --erroronfail
+
+# Create an OpenShift pull secret file
+cat > /etc/crio/openshift-pull-secret <<'EOF'
+$(cat "${PULL_SECRET}")
+EOF
+chmod 600 /etc/crio/openshift-pull-secret
+
+%end
+EOFKS
+```
+
+The kickstart file uses a special [ostreecontainer](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreecontainer)
+directive to pull a `bootc` image from the remote registry and use it to install
+the RHEL operating system.
+
+### Create Virtual Machine
+
+Download a RHEL boot ISO image from https://developers.redhat.com/products/rhel/download.
+Copy the downloaded file to the `/var/lib/libvirt/images` directory.
+
+Run the following commands to create a RHEL virtual machine with 2 cores, 2GB of
+RAM and 20GB of storage. The command uses the kickstart file prepared in the
+previous step to pull a `bootc` image from the remote registry and use it to install
+the RHEL operating system.
+
+```bash
+VMNAME=microshift-4.18-bootc
+NETNAME=default
+
+sudo virt-install \
+    --name ${VMNAME} \
+    --vcpus 2 \
+    --memory 2048 \
+    --disk path=/var/lib/libvirt/images/${VMNAME}.qcow2,size=20 \
+    --network network=${NETNAME},model=virtio \
+    --events on_reboot=restart \
+    --location /var/lib/libvirt/images/rhel-9.4-$(uname -m)-boot.iso \
+    --initrd-inject kickstart.ks \
+    --extra-args "inst.ks=file://kickstart.ks" \
+    --wait
+```
+
+Log into the virtual machine using the `redhat:<password>` credentials.
```

Comment:
Should we preface that paragraph with something along "After install concludes and RHEL boots" or smth?